ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -458,6 +462,54 @@ tests:
     env:
       PROJECT_NAME: machine-config-operator
     workflow: openshift-ci-security
+- as: e2e-aws-ovn-upgrade-ocb
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|OWNERS_ALIASES|LICENSE)$
+  steps:
+    cluster_profile: aws
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+      OCP_ARCH: arm64
+    test:
+    - as: get-prelayering-rpmostree-status
+      commands: |-
+        #!/usr/bin/env bash
+        set -xeuo pipefail
+        run-on-all-nodes --keep-going --exit-zero 'rpm-ostree status'
+      from: zacks-toolbox
+      resources:
+        requests:
+          cpu: 100m
+    - as: enable-layering
+      commands: |-
+        #!/usr/bin/env bash
+        set -xeuo pipefail
+        onclustertesting setup in-cluster-registry --enable-feature-gate --pool=layered
```

Comment:
We want to opt all of the nodes in the cluster into OCL. Instead of doing it this way, use the CI setup mode:

```console
$ onclustertesting setup ci --inject-yum-repos --containerfile-path <(curl -Lo https://raw.githubusercontent.com/openshift/machine-config-operator/refs/heads/master/test/e2e-techpreview/Containerfile.cowsay)
```

This should opt both the control-plane and worker pools into OCL, wait for the process to opt-in process to complete, then proceed further.